articles/101_build_tool.md

```diff
@@ -42,6 +42,50 @@ As a consequence of being developed separately certain features are only availab
 The reason to work on a single universal build tool comes down to reducing the effort necessary for development and maintenance.
 Additionally this makes new features developed for one ROS version / build system available to the other supported ROS versions / build systems.
 
+### Current Workspace and tool combinations
+
+| Concept Layer | ROS2                               | ROS1 (catkin)                  | ROS1 (rosbuild)         |
+|--------------|------------------------------------|--------------------------------|-------------------------|
+| Workspace    | ROS2 ament workspace               | ROS1 catkin workspace          | ROS1 rosbuild workspace |
+| Build tool   | ament_tools                        | catkin_tools, catkin_make, cmi | rosbuild                |
+| Build system | ament_cmake, cmake, python         | catkin, cmake                  | rosbuild                |
+| Package      | ROS2 package, cmake/python project | ROS1 package, cmake project    | ros1 rosbuild package   |
+
+(Note catkin_make cannot build pure cmake projects)
+
+### Phase 1 goal: unify build systems (not rosbuild)
```

Comment:
If someone would like to implement an extension to the unified build tool to support rosbuild that would be absolutely feasible. There this shouldn't be excluded like this.